provider/aws/aws.go

```diff
@@ -166,7 +167,7 @@ func NewAWSProvider(awsConfig AWSConfig) (*AWSProvider, error) {
 		SharedConfigState: session.SharedConfigEnable,
 	})
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "failed to instantiate AWS session")
```

Comment:
Maybe ...

```go
return nil, fmt.Errorf("failed to instantiate AWS session: %w", err)
```

See https://blog.golang.org/go1.13-errors for reference.

Reply:
Yes, I hesitated to introduce a dep to a Go version; but just checked the CI and Dockerfile are using 1.14 so I assume we're good with go >= 1.13. Updated.